lib/flappy_search_bar.dart

```diff
@@ -405,12 +406,45 @@ class _SearchBarState<T> extends State<SearchBar<T>>
                     duration: Duration(milliseconds: _animate ? 1000 : 0),
                     child: AnimatedContainer(
                       duration: Duration(milliseconds: 200),
-                      width:
-                          _animate ? MediaQuery.of(context).size.width * .2 : 0,
+                      width: () {
```

Comment:
May we make a separated method with this logic ?
In order to not make the render method too long

Reply:
oh yeah that is a good idea